tests/test_free_energy.py

```diff
@@ -553,3 +557,46 @@ def test_trajectories_by_replica_to_by_state(seed, n_states, n_iters, n_atoms):
     traj_by_replica = sim_res.extract_trajectories_by_replica(atom_idxs)
     traj_by_state = trajectories_by_replica_to_by_state(traj_by_replica, replica_idx_by_state_by_iter)
     np.testing.assert_array_equal(traj_by_state, frames[:, :, atom_idxs])
+
+
+def test_assert_potentials_compatible(hif2a_ligand_pair_single_topology):
```

Comment:
nit: Think this can be put to `nogpu` marker.

Reply:
Added in https://github.com/proteneer/timemachine/pull/1366/commits/4d1d26f3e81391bd08dd6f02b675acae4fed5cf7